mmtrack/core/evaluation/eval_sot_vot.py

```diff
@@ -152,7 +152,7 @@ def eval_sot_accuracy_robustness(results,
                 - special tracking state: [0] denotes the unknown state,
                     namely the skipping frame after failure, [1] denotes the
                     initialized state, and [2] denotes the failed state.
-        annotations (list[list[dict]]): The first list contains the
+        annotations (list[list[ndarray]]): The first list contains the
```

Comment:
Now, the inputs (`results` and `annotations`)  of `eval_sot_ope`、`eval_sot_eao`、`eval_sot_accuracy_robustness`  are all the same format:  list[list[array]], except that the `annotations` of `eval_sot_ope` is list[dict],  because it has to use the  'visible' information.

Other inputs are all only have tracking bboxes, no other infomation.  I think it may not necessary to add a 'bboxes' key deliberately only for taking care of the 'annotations' of `eval_sot_ope`.

if someone want to evaluate by both ope and vot, he must retest the tracker for recording the tracking states (1 or 2) in tracking results.

Reply:
We need to unify the args of two evalution functions.